packages/form-footer/src/FormFooter.story.tsx

```diff
@@ -4,11 +4,16 @@ import Button from '@leafygreen-ui/button';
 import { css } from '@leafygreen-ui/emotion';
 import Icon from '@leafygreen-ui/icon';
 import FormFooter from '.';
+import defaultArgTypes from '../../../stories/defaultArgTypes';
 
 export default {
   title: 'Components/FormFooter',
   component: FormFooter,
+  args: {
+    darkMode: false,
+  },
   argTypes: {
+    darkMode: defaultArgTypes.darkMode,
```

Comment:
Can we update the controls for the other props while we're in here?
![Screen Shot 2022-09-02 at 2 29 18 PM](https://user-images.githubusercontent.com/2414030/188215891-509177c2-8580-4079-bb83-5c7c0ae76eba.png)

Reply:
Would are you suggesting I change for the `primaryButton` prop? For the other props I can remove what isn't necessary for storybook.

Reply:
I'd maybe do something like we do in MongoNav for the `dataFixtures ` prop

Reply:
oh i see, fill it out a bit